packages/core/src/Config.ts

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
any reason to have this file's name starting with capital letter?

Reply:
It should be because it exports the ConfigService class as default.

@wtrocki if you had concerns about usage from JS code this could be lowercase `config.ts` and avoid using the default export, that way JS could still do

```javascript
let {IServiceConfig, IAeroGearConfig, ConfigService} = require("./config");
```

Reply:
@paolobueno - this could be good style example. In RainCatcher we made style for Uppercase letters. 
I will avoid default export - default exports are cancer :P